conduction-design-tokens/src/component/conduction/icon.tokens.json

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
As discussed in our call: all the _extra_ Utrecht tokens should live inside the `/utrecht` directory, and not in the `/conduction` directory. Keeping them separate from the existing tokens is a great idea though.